starter-s3.yaml

```diff
@@ -143,13 +143,75 @@ data:
     presubmits:
 
     periodics:
-    - interval: 20m
+    - interval: 3h
       agent: kubernetes
       name: echo-test
       spec:
         containers:
         - image: alpine
           command: ["/bin/date"]
+    - cron: "17 * * * *"  # Every hour at 17 minutes past the hour
```

Comment:
did you manually test the job ?

Reply:
We have a Kubernetes Job to test it. It's like a one-shot test.  For example, just in case you want to run the job immediately. How will you do that?

Reply:
I mean following jobs. The parameter are same, right?

https://github.com/kitex-contrib/test-infra/blob/main/job/label_sync_job.yaml

Reply:
> I mean following jobs. The parameter are same, right?

Yes, the parameters in those cronjobs are same with the job in /job/...yaml. However, the peribolos job haven't `run --confirm=true` for worrying about destruction of kitex-contrib org profile...I just run them in dryRun to make sure that  thet wroks in right environment.

BTW, the peribolos needs the PR https://github.com/kitex-contrib/test-infra/pull/36 to be merged.